README.md

```diff
@@ -4,90 +4,87 @@ minideb
-including [mariadb](https://github.com/bitnami/bitnami-docker-mariadb), 
-[redis](https://github.com/bitnami/bitnami-docker-redis), 
-[nginx](https://github.com/bitnami/bitnami-docker-nginx) and 
-[mongodb](https://github.com/bitnami/bitnami-docker-mongodb). 
-
-Compatibility
--------------
-
-The image points to the Debian archive, so you are free to
-install packages from there that you need. However because
-some `Essential` packages have been removed they may not
-always install or work correctly.
-
-In those cases you can figure out which package is needed
-and manually specify to install it along with your desired
-packages.
-
-Docker
-------
+# What is Minideb
+A minimalist Debian based image built specifically to be used as a base image for containers.
 
+# Use Minideb
 You can use the image directly, e.g.
+```
+$ docker run --rm -it bitnami/minideb:latest
+```
 
-    $ docker run --rm -it bitnami/minideb:latest
-
-There are tags for the different Debian releases.
+There are [tags](https://hub.docker.com/r/bitnami/minideb/tags/) for the different Debian releases.
+```
+$ docker run --rm -it bitnami/minideb:jessie
+```
 
-    $ docker run --rm -it bitnami/minideb:jessie
-
-The images are built daily and have the security release enabled,
-so will contain any security updates released more than 24 hours
-ago.
+The images are built daily and have the security release enabled, so will contain any security updates released more than 24 hours ago.
 
 You can also use the images as a base for your own `Dockerfile`:
+```
+FROM bitnami/minideb:jessie
+```
+
+# Why use Minideb
+  * This image aims to strike a good balance between having small images, and having many quality packages available for easy integration.
+  * The image is based on glibc for wide compatibility, and has apt for access to a large number of packages. In order to reduce size of the image, some packages that aren't required in containers are removed:
+    * Packages that aren't often used in containers (hardware related, init systems etc.)
+    * Some files that aren't usually required (docs, man pages, locales, caches)
+  * These images also include an `install_packages` command that you can use instead of apt. This takes care of some things for you:
+    * Install the named packages, skipping prompts etc.
+    * Clean up the apt metadata afterwards to keep the image small.
+    * Retrying if apt fails. Sometimes a package will fail to download due to a network issue, and this may fix that, which is particularly useful in an automated build pipeline.
+
+    For example:
+    ```
+    $ install_packages apache2 memcached
+    ```
+
+# Adoption of Minideb
+The minideb container image is the base image for many Bitnami-maintained language runtimes including
+  [php](https://github.com/bitnami/bitnami-docker-php-fpm),
+  [nodejs](https://github.com/bitnami/bitnami-docker-node),
+  [ruby](https://github.com/bitnami/bitnami-docker-ruby)
+  and infrastructure components
```

Comment:
I think the formatting is wrong here, it should be

|and infrastructure components including
|  [mariadb]...